automatic-off-session-payment/templates/booking-request-accepted/booking-request-accepted-html.html

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Many of our email templates have an empty first line. I don't know that there would be any reason for this. If the templates work fine without the newlines, in my opinion we can leave them out from this process.

Reply:
No need to start updating all processes though, the empty first lines can be removed in those in case we happen to modify them in the future.

Reply:
Tested this, it seems to work fine without the empty first line so I cleaned them up from the templates in this process.